apps/src/templates/instructions/ResourceLink.jsx

```diff
@@ -4,6 +4,8 @@ import color from '@cdo/apps/util/color';
 import BaseDialog from '../BaseDialog';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
+const dialogRatio = 0.8;
```

Comment:
nit: const values at the top should be in the format `DIALOG_RATIO`
https://google.github.io/styleguide/javascriptguide.xml#Constants